release/artifacts.yaml

```diff
@@ -97,9 +97,9 @@
 - target: //ledger/ledger-api-test-tool:ledger-api-test-tool
   type: jar-deploy
 - target: //language-support/scala/codegen:codegen
-  type: jar
+  type: jar-scala
 - target: //language-support/scala/codegen:codegen-main
-  type: jar
+  type: jar-scala
```

Comment:
the codegen jars shouldn't be released as sources. we consider them "binary" tools that users should execute rather than use as a library.

Reply:
👍